### PR TITLE
Update of version command

### LINKED ISSUE #288: Update corectl version command
We should update the information functionality in the version command with the curl suggestion as a last resort.

### PR DESCRIPTION
**Pull request by glooms**
Updating the version command to add feedback about the latest releases and that they can be found on `snap` and eventually on `brew`. The code for brew is commented out as we are not yet available on brew.

The update suggestions are based on the operating system the user is using, thoughts if this is good and should we still include how to curl it?
Also, maybe we should describe how to install the tap on brew if they haven't already?

This closes #288

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -64,47 +64,56 @@ func checkLatestVersion() {
...
-
+		fmt.Println("--------------------------------------------------")
+		fmt.Printf("corectl version: %s, latest version is %s\n", version, res.Current)
+		switch runtime.GOOS {
+			//	case "darwin":
+			//		fmt.Println("To udpate to the latest version using brew just run:")
+			//		fmt.Print("\n  brew upgrade qlik-corectl\n\n")
+			//		fmt.Println("If you prefer using curl you can instead run:")
+		case "linux":
+			fmt.Println("To udpate to the latest version using snap just run:")
```
Comment (sublibra) [inline cmd/utils.go]: typo: udpate

File: cmd/utils.go [hunk 2]
```diff
@@ -64,47 +64,56 @@ func checkLatestVersion() {
...
+			fmt.Print("\n  snap refresh qlik-corectl\n\n")
```
Comment (sublibra) [inline cmd/utils.go]: why a mix of Println and Print with \n?

File: cmd/utils.go [hunk 3]
```diff
@@ -64,47 +64,56 @@ func checkLatestVersion() {
...
+			fmt.Println("If you prefer using curl you can instead run:")
```
Comment (sublibra) [inline cmd/utils.go]: wording: If you prefer curl, you can run:

File: cmd/utils.go (hunk 2 above)
Comment (glooms) [reply cmd/utils.go]: The linter complains about \n in Println ^^

**Pull request merged**